### ISSUE TITLE
Torch 2.0.1+cpu

### ISSUE THREAD

**Issue by haruhix** [vladmandic/automatic#1424]
### Issue Description

after the --upgrade , for some reason, Torch changed to the cpu.
How do i get it back to the gpu?

### Version Platform Description

22:47:24-316415 INFO     Starting SD.Next
22:47:24-318919 INFO     Python 3.10.11 on Windows
22:47:24-360853 INFO     Version: a9f66cb3 Wed Jun 14 12:18:08 2023 +0300
22:47:26-479303 INFO     Latest published version: 0ddf613b49aadafda10f0f84df653d777df3356d 2023-06-14T15:22:59Z
22:47:26-482579 INFO     nVidia CUDA toolkit detected
22:47:27-752280 INFO     Torch 2.0.1+cpu
22:47:27-885442 INFO     Disabled extensions-builtin: ['sd-webui-inpaint-anything', 'sd-webui-txt-img-to-3d-model']
22:47:27-886942 INFO     Enabled extensions-builtin: ['a1111-sd-webui-lycoris', 'clip-interrogator-ext', 'LDSR', 'Lora',
                         'multidiffusion-upscaler-for-automatic1111', 'ScuNET', 'sd-dynamic-thresholding',
                         'sd-extension-aesthetic-scorer', 'sd-extension-steps-animation', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet', 'sd-webui-model-converter', 'seed_travel',
                         'stable-diffusion-webui-images-browser', 'stable-diffusion-webui-rembg', 'SwinIR']
22:47:27-892572 INFO     Disabled extensions: ['sd-webui-inpaint-anything', 'sd-webui-txt-img-to-3d-model']
22:47:27-893322 INFO     Enabled extensions: ['a1111-mini-paint', 'ABG_extension', 'adetailer', 'batch-face-swap',
                         'canvas-zoom', 'infinite-zoom-automatic1111-webui', 'sd-webui-aspect-ratio-helper',
                         'sd-webui-regional-prompter', 'sd-webui-segment-anything', 'sd-webui-stablesr',
                         'sd_civitai_extension', 'Stable-Diffusion-Webui-Civitai-Helper',
                         'stable-diffusion-webui-depthmap-script']
22:47:27-898086 INFO     No changes detected: Quick launch active
22:47:27-903180 INFO     Extension preload: 0.0s Z:\SD\extensions-builtin
22:47:27-903931 INFO     Extension preload: 0.0s Z:\SD\extensions
22:47:27-922320 INFO     Server arguments: []
Z:\SD\venv\lib\site-packages\torchvision\io\image.py:13: UserWarning: Failed to load image Python extension: 'Could not find module 'Z:\SD\venv\Lib\site-packages\torchvision\image.pyd' (or one of its dependencies). Try using the full path with constructor syntax.'If you don't plan on using image functionality from `torchvision.io`, you can ignore this warning. Otherwise, there might be something wrong with your environment. Did you have `libjpeg` or `libpng` installed before building `torchvision` from source?
  warn(

PS C:\Users\axenk> nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2022 NVIDIA Corporation
Built on Wed_Sep_21_10:41:10_Pacific_Daylight_Time_2022
Cuda compilation tools, release 11.8, V11.8.89
Build cuda_11.8.r11.8/compiler.31833905_0

C:\Users\axenk>pip show torch
Name: torch
Version: 2.0.1+cu118
Summary: Tensors and Dynamic neural networks in Python with strong GPU acceleration
Home-page: https://pytorch.org/
Author: PyTorch Team
Author-email: packages@pytorch.org
License: BSD-3
Location: c:\users\axenk\appdata\local\programs\python\python310\lib\site-packages
Requires: filelock, jinja2, networkx, sympy, typing-extensions
Required-by: accelerate, torchaudio, torchvision, xformers

### Acknowledgements

- [X] I have read the above and searched for existing issues

**Comment by vladmandic** [comment]
any updates?

**Comment by haruhix** [comment]
> any updates?

The issue has been resolved. Reinstalling with the parameters helped. 

> webui.bat --use-cuda --reinstall
